yolov5/README.md

```diff
@@ -128,12 +128,16 @@ wget https://github.com/joannzhang00/ImageNet-dataset-classes-labels/blob/main/i
 # Build and serialize TensorRT engine
 ./yolov5_seg -s yolov5s-seg.wts yolov5s-seg.engine s
 
+# Download the labels file
+wget -O coco.txt https://raw.githubusercontent.com/amikelive/coco-labels/master/coco-labels-2014_2017.txt
+
 # Run inference
-./yolov5_seg -d yolov5s-seg.engine ../samples
+./yolov5_seg -d yolov5s-seg.engine ../samples ../coco.txt
```

Comment:
Should this be like this?
./yolov5_seg -d yolov5s-seg.engine ../samples coco.txt